test/api/v3/integration/groups/POST-groups_id_removeMember.test.js

```diff
@@ -80,6 +89,20 @@ describe('POST /groups/:groupId/removeMember/:memberId', () => {
 
       expect(_.findIndex(invitedUserWithoutInvite.invitations.guilds, {id: guild._id})).eql(-1);
     });
+
+    it('sends email to user with rescinded invite', async () => {
+      await leader.post(`/groups/${guild._id}/removeMember/${invitedUser._id}`);
+
+      expect(email.sendTxn).to.be.calledOnce;
+      expect(email.sendTxn.args[0][1]).to.be.eql('guild-invite-rescinded');
+    });
+
+    it('sends email to removed user', async () => {
+      await leader.post(`/groups/${guild._id}/removeMember/${member._id}`);
+
+      expect(email.sendTxn).to.be.calledOnce;
+      expect(email.sendTxn.args[0][1]).to.be.eql('kicked-from-guild');
+    });
```

Comment:
Suggestion: for both of these tests, make sure the email is sent to the correct user (the one who got removed or had their invite rescinded).